en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-400.md

```diff
@@ -20,6 +20,49 @@ Before you begin:
 -   **If you are using Oracle**, commit the changes after running the scripts given below.
     
 ### Preparing for Migration
+
+#### Run the following script on the registry database to add missing registry indices.
```

Comment:
@samgnaniah @Lakith-Rambukkanage since this will be visible as a subtitle on menu, shall we modify it as below ?

```suggestion
#### Creating registry indices

Run the following script on the registry database to add missing registry indices.
```